changelog.md

```diff
@@ -3,6 +3,21 @@
 Any breaking changes to the `topology.yaml` or `shotover` rust API should be documented here.
 This assists us in knowing when to make the next release a breaking release and assists users with making upgrades to new breaking releases.
 
+## 0.2.1
```

Comment:
0.2.0 has not been released yet so this should go under the 0.2.0 heading

Reply:
lets also remove 0.2.1 as its a bit confusing to have it when 0.2.0 still doesnt exist yet.